lcm-derive/src/parse.rs

```diff
@@ -0,0 +1,322 @@
+use quote;
+use syn;
+
+#[derive(Debug)]
+pub struct Field
+{
+	pub name: String,
```

Comment:
Should maybe just be a `syn::Ident` since it never actually is used as a `String`.